src/Root.jsx

```diff
@@ -16,7 +28,7 @@ export default class Root extends Component {
     return (
       <Web3ContextProvider>
         <Provider store={store}>
-          <BrowserRouter basename={"/#"}>
+          <BrowserRouter>
```

Comment:
What happens for people who have it bookmarked with the `/#` can we add redirects for now?

Reply:
ok, https://github.com/OlympusDAO/olympus-frontend/pull/372/commits/5dd6ad555a9a85dcafd75d7e813322bff39bb953, restores this functionality since it was needed for IPFS